pages/common/gcloud-init.md

```diff
@@ -0,0 +1,20 @@
+# gcloud init
+
+> This command launches an interactive workflow to setup gcloud or reinitialize configurations.
+> More information: <https://cloud.google.com/sdk/gcloud/reference/init>.
+
+- Launch a "Getting Started" workflow:
+
+`gcloud init`
+
+- Launch a workflow without diagnostics:
+
+`gcloud init --skip-diagnostics`
+
+- Allow using the console for authentication:
+
+`gcloud init --console-only`
+
+- Do not launch a browser:
+
+`gcloud init --no-launch-browser`
```

Comment:
This does exactly the same thing as `--console-only`.
https://cloud.google.com/sdk/gcloud/reference/init
> --console-only, --no-launch-browser
    Prevent the command from launching a browser for authorization. Use this flag if you are on a machine that does not have a browser and you cannot install the gcloud CLI on another machine with a browser. 
```suggestion
```